src/camvidlog2/moondream/cli.py

```
@@ -0,0 +1,88 @@
+    output: Path,
+    db: Annotated[Path, typer.Option()] = Path("tmp.feather"),
+):
+    # load existing array, if any
+    array = data_load(db)
+    if array is None:
+        raise click.FileError("database file must exist")
+
+    video = video.resolve()
+    array = array.loc[array.index.get_level_values("filename") == str(video)]
+    if array.shape[0] == 0:
+        raise ValueError("video must be in database")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)